### ISSUE TITLE
Can you provide the setup for training?

### ISSUE THREAD

**Issue by Taeuk-Jang** [ecreager/laftr#1]
Thank you for the awesome code.

I would like to run the model with adult dataset, but I am not sure how to tune the hyper parameter and how long should I train. 
Could you provide the data for that?

And also how do you pre-processed the data? I would also like to try for compas and german but I am not sure how you pre-processed.

Thank you in advance.

**Comment by ecreager** [comment]
Check out appendix B in the arxiv version (https://arxiv.org/pdf/1802.06309.pdf) for training details including hyperparameter ranges and model selection criteria for the Pareto fairness-accuracy tradeoff curves. We didn't run experiments on the Compas or German datasets but I expect that measuring the loss on a validation set to choose hyperparameters should suffice for hyperparameter selection. Or sweep over a large range and choose best-case models to produce the Pareto curves if that's what you're looking for.
